app/dashboard/_components/UploadThingModal.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { setAppSetting } from '~/actions/appSettings';
+import { UploadThingTokenForm } from '~/app/(blobs)/(setup)/_components/UploadThingTokenForm';
+import Link from '~/components/Link';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '~/components/ui/dialog';
+import Paragraph from '~/components/ui/typography/Paragraph';
+
+function UploadThingModal() {
+  return (
+    <Dialog open={true}>
```

Comment:
Forcing this to open means that the user can't use the app until they fix this issue. While it is important that it be fixed, the app can still be used without adding the token. This should be closable.